### ISSUE TITLE
t/32-was_locked.t fails since 2016-11-12

### ISSUE THREAD

**Issue by eserte** [MediaWiki-Bot/MediaWiki-Bot#81]
It seems that there are only FAIL reports since some days, see http://matrix.cpantesters.org/?dist=MediaWiki-Bot%205.006003;reports=1#sl=7,1

On my smoker systems the test log looks like this:
```
...
#   Failed test 'lock history'
#   at t/32-was_locked.t line 16.
# Looks like you failed 1 test of 2.
t/32-was_locked.t .............. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/2 subtests
...
```

**Comment by afonsoguerra** [comment]
I can confirm this is also happening on my system.

**Comment by jkeenan** [comment]
Also present when I attempted to install against perl-5.27.10 on FreeBSD-11.0.

**Comment by doherty** [comment]
Please see [`CONTRIBUTING.md`](https://github.com/MediaWiki-Bot/MediaWiki-Bot/blob/master/CONTRIBUTING.md)

**Comment by wp-seth** [comment]
should be fixed in #92.
